### DISCUSSION TITLE
关于大海捞针测试的问题

### DISCUSSION THREAD

**Discussion by abc123456cxx** [open-compass/opencompass#919] [Q&A] [answered]
看config中cdme数据集配置时有一个'tokenizer_model': 'gpt-4'参数，想请问一下这里的这个参数是什么作用呢，如果要测试自己的模型也要用gpt-4嘛 还是需要修改成自己模型tokenizer的路径呢？感谢回答
![2024-02-26 14-40-08屏幕截图](https://github.com/open-compass/opencompass/assets/150245861/e761b11c-da1a-41fc-b853-a87d573d03ce)

**Comment by tonysy** [comment, marked as answer]
We use it for length calculation(token).

**Reply by abc123456cxx** [reply]
那请问一下如果想测试自己的模型的时候需要修改嘛

**Reply by tonysy** [reply]
No

**Reply by abc123456cxx** [reply]
好的好的感谢
